process/incident_management.md

```diff
@@ -0,0 +1,263 @@
+    1. First Responder opens the bridge call for that
+3. The Incident Commander manages the communication and coordination around the recovery
+    1. By creating a dedicated IM channel to record the audit trail of the incident resolution.
+    2. Post a notification in the relevant channels for external awareness.
+    3. By involving the right people to help and speed up the recovery process
+    4. By ensuring a good communication with first level support and the customer
+4. The incident is recovered
+5. The Incident Owner (usually the First Responder unless the Incident Commander picked someone else) is responsible for running and documenting the post mortem and root cause analysis and bring the incident to full closure.
+
+## Incidents
+An incident is an event that can occur at any time and which results in a degradation or outage of one or multiple of our services.
+
+An incident can be raised by a customer or directly by our monitoring and alerting systems, or directly by a member of the SRE team.
+
+Depending on the impact of the incident on the services and customers it will be categorized in terms of severity.
+
+As an SRE organization we should aim at:
+
+* Minimizing the time it takes us to recover the service (MTTR)
+* Improve Reliability of our services: i.e. having as few incidents as possible, optimizing prevention of incidents
+* Having incidents with the least impact to our customers
+* Catching incident occurrences through our automation (monitoring/alerting) before they are noticed by the customer
+* Avoiding re-occurrences of similar incidents
+* Minimizing the time to detect a potential incident (MTTA)
+
+## Roles and Responsibilities
+
+### Incident First Responder
+Any SRE investigating a cluster issue becomes this, when they notice a **problem with a cluster or a specific application**  which:
+* may be a **customer–facing cluster having an issue impacting (degradation or outage)**
+* or cause **imminent major outage.**
+
+A major outage can be defined as one where our platform is running out of error budget and we start failing our SLOs.
+
+SRE teams have the concept of an on-duty shift person who is responsible for answering to alerts and incidents while the rest of the team is focusing on engineering work. These persons will usually be First Responders. Most often a shift Primary or Secondary become First Responders while investigating ongoing cases/alerts.
+
+
+#### Initial Response
+Before doing any further troubleshooting, First Responder must, in order:
+
+1. **Create a bridge.**
+2. Using IM, **post a message in the relevant channels** with:
+    * link to bridge,
+    * short description of issue (which service, what’s failing),
+    * your IM handle.
+Only now can you continue debugging while describing the problem to people joining the bridge.
+
+#### 10 Minutes Later
```

Comment:
I find this section too elaborate for a public document. We could keep this privately, and not hardcode any timelines. It's up to every team (meaning outside RH) who reads this doc to define their own escalation path